openapi-templates/model_utils.mustache

```diff
@@ -1714,3 +1715,14 @@ def validate_get_composed_info(constant_args, model_args, self):
       additional_properties_model_instances,
       discarded_args
     ]
+
+
+def async_wrap(func):
```

Comment:
Are these .mustache files fully auto-generated by OpenAPI Generator?

Reply:
These `.mustache` files come from here, I've just pulled them from this repo and made modifications to them: https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources/python